packages/block-editor/src/components/global-styles/utils.js

```diff
@@ -17,6 +17,8 @@ export const ROOT_BLOCK_SUPPORTS = [
 	'color',
 	'linkColor',
 	'buttonColor',
+	'headingColor',
+	'headingBackgroundColor',
```

Comment:
do you know why the button is missing `buttonBackgroundColor` here? could we use just the one?

Reply:
By Riad's suggestion, I take it that using only one would work,
but I would have to refer to @scruffian if this is even the correct method to hide the options for the individual blocks.